src/main/java/sg/ncl/MainController.java

```diff
@@ -877,8 +878,7 @@ private String getTeamIdByName(String teamName) throws WebServiceRuntimeExceptio
                 }
 
             } else {
-                JSONObject object = new JSONObject(responseBody);
-                return object.getString("id");
+                return extractTeamInfo(responseBody.toString());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) "responseBody" is already a string, there's no need to call "toString()" on it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1858)